notebooks/remote_functions/remote_function_usecases.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    @bpd.remote_function()
</pre><p>Let's add a comment here explaining this is for demonstration purposes only and for this use case one should prefer using case_when. (<a href="https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.series.Series#bigframes_series_Series_case_when" target="_blank">https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.series.Series#bigframes_series_Series_case_when</a>)</p><p> </p><p>For your reference:</p>

<pre class="ql-syntax" spellcheck="false">duration_cat = df['duration_minutes'].case_when(
     caselist=[
         (df['duration_minutes'].lt(90), "short"),
         (df['duration_minutes'].lt(180), "medium"),
         (True, "long"),
    ]
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Ran into couple of issues (b/349926559, b/349927177), but noted as a broad comment that the use cases are only for demo purpose of remote_function and not necessarily the best way to achieve the end result.</p>